incremental-replication-between-clusters.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Incremental data replication between TiDB clusters in real time
+summary: Learns how to replicate incremental data from one TiDB cluster to another cluster in real time
+---
+
+# Incremental data replication between TiDB clusters in real time
+
+This document describes how to configure a TiDB cluster and its downstream MySQL or downstream TiDB, and then replicate the incremental data from an upstream cluster to a downstream cluster in real time .
+
+## Usage scenarios
+
```

Comment:
```suggestion
```

Reply:
> 

Do we need to remove this subtitle, unlike the Doc-cn PR?

Reply:
Yes, we do. See https://github.com/pingcap/docs-cn/pull/7040.

Reply:
Thanks!